Modules/services/Controller/ServicesprojectsController.php

```diff
@@ -632,7 +641,8 @@ public function kanbanAction($id_space, $id_project) {
             "projectServices" => json_encode($services),
             "projectUsers" => json_encode($projectUsers),
             "mainUser" => $projectMainUser,
-            "personInCharge" => $project['in_charge']
+            "personInCharge" => $project['in_charge'],
+            "userIsManager" => json_encode($isManager),
```

Comment:
why json encode a boolean?

Reply:
if I don't encode (that's what I tried first)  and get it via {{ userIsmanager }} in vuejs instance, it is equal to 1 if true and equal to "" (leads to a bug) if false.
So, I had to json_encode it to keep its type as boolean.

Reply:
if I don't encode (that's what I tried first)  and get it via {{ userIsmanager }} in vuejs instance, it is equal to 1 if true and equal to "" (leads to a bug) if false.
So, I had to json_encode it to keep its type as boolean.